MetaMask/metamask-extension
# UX Multichain: updated assets to tokens in home page

This PR is to update Tab title from Assets to Tokens to keep in sync with Mobile

## Screenshots/Screencaps


### Before

![Screenshot 2023-05-15 at 8 28 12 PM](https://github.com/MetaMask/metamask-extension/assets/39872794/ddf72340-8e81-41f9-b2e9-1be9bc0871cf)


### After

![Screenshot 2023-05-15 at 8 27 53 PM](https://github.com/MetaMask/metamask-extension/assets/39872794/f50ce9fe-5713-45d0-b86e-cea1a7fe55ca)


## Manual Testing Steps

- Go to home screen
- Check first tab has title "Tokens"


## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
